Groups.xml

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="Groups" verbose="1" >
+    <test name="parserNoMethods" >
+        <groups>
+            <run>
+                <include name="checkinTests"  />
+            </run>
+        </groups>
+        <classes>
+            <class name="parser.JsonParserTest"></class>
```

Comment:
collapse empty tag
`<class name="parser.JsonParserTest"/>`
the same for all class tags